Components/CustomConfig/CustomConfig.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace MollieShopware\Components\CustomConfig;
+
+class CustomConfig
+{
+
+    /**
+     * main node for a custom mollie configuration
+     */
+    const KEY_MOLLIE_CONFIG = 'mollie';
+
+    /**
+     * @var array
+     */
+    private $config;
+
+
+    /**
+     * CustomConfig constructor.
+     * 
+     * @param array $shopwareConfig
+     */
+    public function __construct($shopwareConfig)
+    {
+        # let's verify if we have a custom config for mollie
+        # with that custom config its possible to use a different
+        # options for developers that might not be possible in 
+        # the default plugin
+        if (isset($shopwareConfig[self::KEY_MOLLIE_CONFIG])) {
+            $this->config = $shopwareConfig[self::KEY_MOLLIE_CONFIG];
+        } else {
+            $this->config = array();
```

Comment:
why not declaring the empty array in line 16 when defining the $config class param?

Reply:
no fan ob decentralised assignment in this case
because the constructor would only assign 50% of the truth :)